lib/Dialect/ArithExt/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_subdirectory(IR)
+add_mlir_dialect_library(MLIRArithExt
```

Comment:
It would be nice if the prefix was consistent across all CMake targets, currently it seems like some are prefixed with `MLIR` and some with `MLIRHEIR`. Maybe we just want to go for `HEIR`?

```suggestion
add_mlir_dialect_library(HEIRArithExt
```

Reply:
I'd just go with what BUILD / Bazel is using so we are consistent with the products of the build

Reply:
resolved in PR updated on Jul 17th, 11PM (PST)